pkg/mls/store/queries/queries.sql.go

```diff
@@ -420,6 +421,7 @@ func (q *Queries) InsertWelcomeMessage(ctx context.Context, arg InsertWelcomeMes
 		arg.Data,
 		arg.InstallationKeyDataHash,
 		arg.HpkePublicKey,
+		arg.GroupRefreshStateCursor,
```

Comment:
Did you manually edit this file? I don't see the corresponding change in `queries.sql`. This file is generated.

In any case, you will need to update the postgres function for inserting a welcome message. @richardhuaaa wrote the function and can help there.

I would recommend creating a new function `insert_welcome_message_v2` rather than replacing the existing function, so that it doesn't cause errors during deployment.